### ISSUE TITLE
Incompatible color attachments under wayland

### ISSUE THREAD

**Issue by flash-freezing-lava** [mvlabat/bevy_egui#192]
# Relevant system information
OS: Arch Linux
Desktop Environment: GNOME 44.3 using wayland
```
AdapterInfo { name: "NVIDIA GeForce GTX 1070", vendor: 4318, device: 7041, device_type: DiscreteGpu, driver: "NVIDIA", driver_info: "535.54.03", backend: Vulkan }
```

# Description

When compiling the [simple example](https://github.com/mvlabat/bevy_egui/blob/main/examples/simple.rs) with feature `bevy/wayland` activated and running under wayland, it gives a `wgpu error: Validation Error` with the following cause:

```
Caused by:
    In a RenderPass
      note: encoder = `<CommandBuffer-(0, 1, Vulkan)>`
    In a set_pipeline command
      note: render pipeline = `egui render pipeline`
    Render pipeline targets are incompatible with render pass
    Incompatible color attachments at indices [0]: the RenderPass uses textures with formats [Some(Bgra8UnormSrgb)] but the RenderPipeline uses attachments with formats [Some(Bgra8Unorm)]
```

See https://github.com/bevyengine/bevy/discussions/9163 for discussion on the underlying cause.

**Comment by seabassjh** [comment]
I think this is related to #194

**Comment by mvlabat** [comment]
Hi. Is the issue still actual? I don't have a way to reproduce it with wayland, but I hope it was fixed together with #194

**Comment by flash-freezing-lava** [comment]
Since bevy_egui 0.24.0, this error does no longer occur.
